src/PantheonTerminalWindow.vala

```diff
@@ -166,30 +154,37 @@ namespace PantheonTerminal {
                 open_tabs ();
             }
 
-            /* Actions and UIManager */
-            main_actions = new Gtk.ActionGroup ("MainActionGroup");
-            main_actions.set_translation_domain ("pantheon-terminal");
-            main_actions.add_actions (main_entries, this);
+            main_actions = new GLib.SimpleActionGroup ();
+            main_actions.add_action_entries (main_entries, this);
 
             clipboard = Gtk.Clipboard.get (Gdk.Atom.intern ("CLIPBOARD", false));
             update_context_menu ();
             clipboard.owner_change.connect (update_context_menu);
 
             primary_selection = Gtk.Clipboard.get (Gdk.Atom.intern ("PRIMARY", false));
 
-            ui = new Gtk.UIManager ();
+            menu = new Gtk.Menu ();
 
-            try {
-                ui.add_ui_from_string (ui_string, -1);
-            } catch (Error e) {
-                error ("Couldn't load the UI: %s", e.message);
-            }
+            var copy_menuitem = new Gtk.MenuItem.with_label (_("Copy"));
+            var paste_menuitem = new Gtk.MenuItem.with_label (_("Paste"));
+            var select_all_menuitem = new Gtk.MenuItem.with_label (_("Select All"));
+            var search_menuitem = new Gtk.MenuItem.with_label (_("Search"));
+            var show_in_file_browser_menuitem = new Gtk.MenuItem.with_label (_("Show in File Browser"));
 
-            Gtk.AccelGroup accel_group = ui.get_accel_group ();
-            add_accel_group (accel_group);
+            copy_menuitem.activate.connect (action_copy);
```

Comment:
Instead of doing this, we should set `action_name` on the menuitems. This will handle sensitivity